custom_components/tahoma/climate_devices/atlantic_heat_recovery_ventilation.py

```diff
@@ -35,25 +33,21 @@
 COMMAND_REFRESH_VENTILATION_CONFIGURATION_MODE = "refreshVentilationConfigurationMode"
 
 IO_AIR_DEMAND_MODE_STATE = "io:AirDemandModeState"
-
-FAN_MODE_TO_TAHOMA = {
-    FAN_AUTO: "auto",
-    FAN_BOOST: "boost",
-    FAN_KITCHEN: "high",
-    FAN_AWAY: "away",
+IO_VENTILATION_MODE_STATE = "io:VentilationModeState"
+IO_VENTILATION_CONFIGURATION_MODE_STATE = "io:VentilationConfigurationModeState"
+
+TAHOMA_TO_FAN_MODES = {
+    "auto": FAN_AUTO,
+    "away": FAN_BOOST,
+    "boost": FAN_KITCHEN,
+    "high": FAN_AWAY,
 }
 
-TAHOMA_TO_FAN_MODE = {v: k for k, v in FAN_MODE_TO_TAHOMA.items()}
+FAN_MODES_TO_TAHOMA = {v: k for k, v in TAHOMA_TO_FAN_MODES.items()}
 
 HVAC_MODES = [HVAC_MODE_FAN_ONLY]
 PRESET_MODES = [PRESET_AUTO, PRESET_PROG, PRESET_MANUAL]
-FAN_MODES = [FAN_AUTO, FAN_BOOST, FAN_KITCHEN, FAN_AWAY, FAN_BYPASS]
-
-date=date.today()
-DAY=date.day
-MONTH=date.month
-
-VENTILATION_MODE_STATE = {"dayNight": "night","month": MONTH,"test": "off","endOfLineTest": "off","leapYear": "off","day": DAY,"cooling": "off","prog": "off"}
+FAN_MODES = [FAN_AUTO, FAN_BOOST, FAN_KITCHEN, FAN_AWAY]
```

Comment:
FAN_BYPASS mode is missing in the FAN_MODES list

Reply:
Fixed!